### PR TITLE
Update readme with Baltimore cert update info

### PR DESCRIPTION
**Pull request by sharmasejal**
## Checklist
- [x] I have read the [contribution guidelines](https://github.com/Azure/azure-iot-sdk-csharp/blob/master/.github/CONTRIBUTING.md).
- [x] I added or modified the existing tests to cover the change (we do not allow our test coverage to go down).
- [x] This pull-request is submitted against the `master` branch.

### CODE REVIEW

File: readme.md [hunk 1]
```diff
@@ -153,6 +153,25 @@ This repository contains [provisioning service client SDK](./provisioning/servic
 
 > Device Explorer is no longer supported. A replacement tool can be found [here](https://github.com/Azure/azure-iot-explorer).
 
+## Certificates -  Important to know
+
+The Azure IoT Hub certificates presented during TLS negotiation shall be always validated using the appropriate root CA certificate(s).
+
+The samples in this repository leverage the certificates in `certs.c` for the United States, Germany sovereign cloud and China sovereign cloud.
```
Comment (vinagesh) [inline readme.md:160]: This specific sentence is not for our sdk. certs.c is not in our samples. Also samples for .net are in another repository.

Comment (vinagesh) [reply readme.md:160]: @abhipsaMisra - Is there any example Sejal point to for this?

**Review by vinagesh — APPROVED** [review]

File: readme.md (hunk 1 above)
Comment (sharmasejal) [reply readme.md]: updated

File: readme.md [hunk 2]
```diff
@@ -153,6 +153,21 @@ This repository contains [provisioning service client SDK](./provisioning/servic
 
 > Device Explorer is no longer supported. A replacement tool can be found [here](https://github.com/Azure/azure-iot-explorer).
 
+## Certificates -  Important to know
+
+The Azure IoT Hub certificates presented during TLS negotiation shall be always validated using the appropriate root CA certificate(s).
+
+Always prefer using the local system's Trusted Root Certificate Authority store instead of hardcoding the certificates (i.e. using certs.c such as our samples require in certain combinations).
```
Comment (abhipsaMisra) [inline readme.md:160]: "i.e. using certs.c such as our samples require in certain combinations" is not relevant to us

File: readme.md (hunk 1 above)
Comment (abhipsaMisra) [reply readme.md]: No, afaik we don't hardcode certificates anywhere in our samples -> one of our samples actually points to the c repo: https://github.com/Azure-Samples/azure-iot-samples-csharp/tree/master/provisioning/Samples/device/X509Sample

**Review by abhipsaMisra — APPROVED** [review]

**Pull request merged**